modules/update-mirror-repository.adoc

```diff
@@ -9,7 +9,7 @@ Before you upgrade a cluster on infrastructure that you provision in a restricte
 
 .Procedure
 
-. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions.
+. Use the link:https://access.redhat.com/labs/ocpupgradegraph/update_channel[OpenShift Upgrade Graph visualizer and update planner] to plan an upgrade from one version to another. The OpenShift Upgrade Graph provides channel graphs and a way to confirm that there is an update path between your current and intended cluster versions
```

Comment:
Missing a period

Reply:
Actually, should the title of it be "Red Hat OpenShift Container Platform Update Graph" like is shown when you look at the app?

Reply:
Good point; it probably should match the app exactly. I just pushed that update :+1: